src/pyroboplan/core/utils.py

```diff
@@ -151,8 +204,10 @@ def get_random_collision_free_state(model, collision_model, padding=0.0, max_tri
             The model from which to generate a random state.
         collision_model : `pinocchio.Model`
             The model to use for collision checking.
-        padding : float or array-like, optional
+        joint_padding : float or array-like, optional
```

Comment:
in radians?

Reply:
not necessarily, could be prismatic joints

Reply:
Oh right. The world is bigger than my own...